app/rdf/query-search.ts

```diff
@@ -169,6 +169,8 @@ const mkScoresQuery = (
       visualize:hasDimension ?dimensionIri.
 
       ?dimensionIri visualize:hasTermset ?termsetIri .
+      ?dimensionIri visualize:hasTimeUnit ?unitType .
```

Comment:
Since we construct the query, I want to link the dimension with time unit in one hop to facilitate the parsing (`?dimension cubeMeta:dataKind/time:unitType` is two hops).

Reply:
Ah I see, misunderstood its purpose 👍